server/sonar-server/src/main/java/org/sonar/server/issue/ws/BulkChangeAction.java

```diff
@@ -110,16 +110,18 @@
   private final NotificationManager notificationService;
   private final List<Action> actions;
   private final IssueChangeTrigger issueChangeTrigger;
+  private final LiveMeasureComputer liveMeasureComputer;
 
   public BulkChangeAction(System2 system2, UserSession userSession, DbClient dbClient, IssueStorage issueStorage, NotificationManager notificationService, List<Action> actions,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00107)